src/main/java/com/pingcap/tikv/types/DateTimeType.java

```diff
@@ -17,18 +17,15 @@
 
 package com.pingcap.tikv.types;
 
-import com.pingcap.tikv.codec.CodecDataInput;
-import com.pingcap.tikv.codec.CodecDataOutput;
-import com.pingcap.tikv.codec.InvalidCodecFormatException;
 import com.pingcap.tikv.meta.TiColumnInfo;
 
-import java.sql.Timestamp;
-import java.time.LocalDateTime;
 import java.time.ZoneId;
-import java.time.ZonedDateTime;
 
 public class DateTimeType extends TimestampType {
-  private final ZoneId defaultZone = ZoneId.systemDefault();
+  static {
+    defaultZone = ZoneId.systemDefault();
```

Comment:
This share the same static ref with TimestampType. You change it one place, all affected. In other words, this won't work correctly.
You might either use a method and override it for timezone. For classname in warning message you can use this.getClass().

Reply:
you're right, I will change it.

Reply:
fixed